### PR TITLE
Updated pystac version to 1.0.0

### LINKED ISSUE #112: Update to STAC 1.0.0
More information in this blogpost: https://medium.com/radiant-earth-insights/spatiotemporal-asset-catalog-stac-1-0-0-rc-1-released-59e7946b3873

### PR DESCRIPTION
**Pull request by shiwakotisurendra**
Closes #112

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -9,9 +9,12 @@ Fixed
 ^^^^^
 something was fixed
 
+[Master] (2021-**-**)
+---------------------
+
 Changed
 ^^^^^^^
-something was changed
+-``requirements``: Pystac version updated to 1.0.0 #147
```
Comment (fwfichtner) [inline CHANGELOG.rst:17]: Please move down to `[Master]`

File: requirements.txt [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
-requests~=2.25.1
-sentinelsat>=1.0.1
-shapely~=1.7.1
-pydantic
+numpy
+pillow
+pystac[validation]==1.0.0
+python-dateutil
+requests
+sentinelsat>=0.14
```
Comment (fwfichtner) [inline requirements.txt:7]: I guess something went wrong when you resolved the conflict, can you update the version again? `pydantic` is also missing.

**Review by fwfichtner — CHANGES_REQUESTED** [review]
Good job, please go through the comments before we merge.

File: requirements.txt (hunk 2 above)
Comment (shiwakotisurendra) [reply requirements.txt:7]: ok , i will do that

File: CHANGELOG.rst (hunk 1 above)
Comment (shiwakotisurendra) [reply CHANGELOG.rst:17]: Sure

**Review by fwfichtner — APPROVED** [review]

**Pull request merged**